systemtest/src/test/java/io/strimzi/systemtest/watcher/AbstractNamespaceST.java

```diff
@@ -4,67 +4,326 @@
+import static io.strimzi.systemtest.Constants.MIRROR_MAKER2;
+import static io.strimzi.systemtest.enums.CustomResourceStatus.Ready;
 import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
 
+/**
+ * Test suite that implements exact tests to be executed with Cluster Operator set to watch all or multiple namespaces, based on extending class.
+ */
 @IsolatedSuite
 public abstract class AbstractNamespaceST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(AbstractNamespaceST.class);
 
-    static final String SECOND_NAMESPACE = "second-namespace-test";
-    static final String MAIN_NAMESPACE_CLUSTER_NAME = "my-cluster";
-    static final String SECOND_CLUSTER_NAME = MAIN_NAMESPACE_CLUSTER_NAME + "-second";
+    // namespace for all resources in this test except for KafkaTopics and KafkaUsers watched by Primary Kafka Cluser.
+    static final String MAIN_TEST_NAMESPACE = "main-test-namespace";
 
-    private static final String TOPIC_EXAMPLES_DIR = Constants.PATH_TO_PACKAGING_EXAMPLES + "/topic/kafka-topic.yaml";
+    // namespace watched by Primary Kafka Cluster's TO and UO for any KafkaTopic and KafkaUser CRs.
+    static final String PRIMARY_KAFKA_WATCHED_NAMESPACE = "primary-kafka-watched-namespace";
 
-    void checkKafkaInDiffNamespaceThanCO(String clusterName, String namespace) {
-        String previousNamespace = cluster.setNamespace(namespace);
-        LOGGER.info("Check if Kafka {}/{} is READY", namespace, clusterName);
+    // name of kafka cluster which is to be created before any of these tests
+    static final String PRIMARY_KAFKA_NAME = "primary-kafka";
 
-        KafkaUtils.waitForKafkaReady(namespace, clusterName);
+    static String scraperPodName = null;
 
-        cluster.setNamespace(previousNamespace);
+    /**
+     * @description This test case verifies that Kafka (including components Cruise Control and Exporter) deployed in namespace different from one where Cluster Operator
+     * is deployed, is still deployed correctly.
+     *
+     * @steps
+     *  1. - As part of setup, in the main namespace, which is different from Cluster Operator namespace, ephemeral Kafka Cluster with 3 replicas is deployed, also including Kafka Exporter and Cruise Control.
+     *     - Kafka and its components are deployed and ready.
+     *
+     * @usecase
+     *  - namespaces
+     *  - cluster-operator-watcher
+     *  - kafka
+     */
+    @IsolatedTest
+    final void testDeployKafkaWithOperandsInNamespaceDifferentFromCO() {
+        LOGGER.info("Verifying that Kafka: {}/{} ,its Kafka Exporter and Cruise Control are in ready state", MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME);
+        KafkaUtils.waitForKafkaReady(MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME);
```

Comment:
Wait for Kafka readiness is done during the `createResource` method, so I guess you can remove this and keep the asserts